Finished Quest03

RohMoon opened the pull request:
# PullRequest 
## 2021- 12-16  Quest 03.
 - Fin Quest 03.
 - #### Auth Roh Moon

RohMoon committed:
Finished Quest03

nunch-dev commented on Quest03/skeleton/quest.html:
```diff
@@ -33,15 +33,33 @@
+				})
+
 			}
+			let style = document.createElement('style');
+			style.type = 'text/css';
+			style.innerHTML = '1stRowSquare{ background-color:yellow;}';
```
무슨 용도로 사용?

실제 HTML에 어떻게 붙는지 확인해볼것

nunch-dev reviewed (commented):
https://user-images.githubusercontent.com/8156542/146378252-60ad2d20-c114-4774-baac-9cfee69e18bb.mov

정상적으로 동작하지 않음
요구사항을 다시한번 확인하고, 그에 맞게 개발 후 테스트 해보고 올릴 것!

# 수정사항

1.  Quest 3-1은 개별 파일로 뺄 것.
2. Quest 3-2는 다음과 같은 요구사항에 맞게 만들 것.
    * 사각형 각각 독립적으로 배경색이 변해야 함
    * 첫째 줄의 사각형과 둘째 줄의 사각형은 독립적으로 동작해야 함
3. 모든 상황에서 다음을 지킬 것
    * 변수에 재할당이 없는 경우 반드시 `const`를 사용할 것.
    * 변수에 재할당이 되는 경우 반드시 `let`을 사용할 것.
    * 어떠한 경우에서도 `var`는 절대 사용 금지

# 기타
`Quest`와 `Advanced` 를 구분할 것

RohMoon committed:
Finished Quest03

Modifiy
 - Adevanced 에서 `var`로 변경시 코드 변화점 확인 후 `let`으로 변경 다시 복원 하지 않아 발생한 부분인점 확인됨.

 - let `stlye` 선언 후 `class` 를 추가하는 방법으로  Quest01 해결하려했던 과정에서 삭제되지않은 코드 삭제.

 - quest03-1 js 파일 추가.
